dash/_hooks.py

```diff
@@ -0,0 +1,222 @@
+import typing as _t
```

Comment:
just curious: why this rather than importing the specific things you need from `typing` (e.g., `from typing import TypeVar`)?